.github/workflows/stalebot.yml

```diff
@@ -12,27 +12,33 @@ jobs:
     stale:
         runs-on: ubuntu-latest
         permissions:
-            contents: read
+            contents: write
             issues: write
             pull-requests: write
         if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
         steps:
-            - name: Process stale issues
+            - name: Process stale issues and PRs
               uses: actions/stale@v9.0.0
               with:
                   repo-token: ${{ secrets.GITHUB_TOKEN }}
-                  stale-issue-message: "As a part of this repository's maintenance, this issue is being marked as stale due to inactivity. Please feel free to comment on it in case we missed something.\n\n###### After 7 days with no activity this issue will be automatically be closed."
+                  stale-issue-message: "As a part of this repository's maintenance, this issue is being marked as stale due to inactivity. Please feel free to comment on it in case we missed something.\n\n###### After 7 days with no activity this issue will be automatically closed."
                   close-issue-message: 'This issue was closed because it has been 14 days with no activity.'
                   operations-per-run: 140
                   days-before-stale: -1
                   days-before-close: -1
                   days-before-issue-stale: 7
                   days-before-issue-close: 7
                   stale-issue-label: 'status: stale'
-                  stale-pr-label: 'status: stale'
                   exempt-issue-labels: 'type: enhancement'
                   only-issue-labels: 'needs: author feedback'
                   close-issue-label: "status: can't reproduce"
+                  days-before-pr-stale: 30
+                  days-before-pr-close: 14
+                  stale-pr-label: 'status: stale'
+                  only-pr-labels: 'needs: author feedback,type: community contribution'
```

Comment:
Should we only address community contributions? I cannot find a reason not to apply this to any PR.

Reply:
You're right. I have excluded that requirement. Makes sense to include other PRs as well.

Reply:
Cool, thanks for this. I think we should remove the `needs: author feedback` label as well though. Not sure how/if that label applies to non-community PRs.

Reply:
While I agree in principle, the truth is that we don't do such a good job at reviewing PRs (particularly from the community) and there are a few that we've only reviewed after months or even years. Or that remain in this state after some updates from the author. That's why I thought it'd be safer to apply stalebot only to those PRs where we've taken a look and responded and are really waiting for something to happen.

Reply:
WFM!